airbyte-webapp/src/views/layout/SideBar/components/SidebarDropdownMenu.module.scss

```diff
@@ -0,0 +1,78 @@
+@use "../../../../scss/colors";
+@use "../../../../scss/variables";
+
+.sidebarMenu {
+  position: relative;
+}
+
+.button {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  justify-content: center;
+  height: 70px;
+  width: 100%;
+  background: transparent;
+  border: 0;
+  outline: none;
+  cursor: pointer;
+  border-radius: variables.$border-radius-md;
+  line-height: 15px;
```

Comment:
Is this needed? Trying to get away from custom text styles.

Reply:
But It's the current design in the sidebar. 
![image](https://user-images.githubusercontent.com/66960359/191030786-81fe150f-1ef4-48a1-84e6-f4bced9c5241.png)

Reply:
Some properties removed (`line-height`)

Reply:
Also, we have a design where `font-size: 9px` for this component.
![image](https://user-images.githubusercontent.com/66960359/191067834-f5b4d257-c7a2-4ace-a3ce-0e27a06465d4.png)

Reply:
@edmundito shared some thoughts about it, that we should keep the custom design in the sidebar

Reply:
Yes, should keep the custom design but do assign the size to a variable in the scss file so that we know it's intentionally different.